backend/app/routes/auth_routes.py

```
@@ -0,0 +1,104 @@
+        username=data['username'],
+        email=data['email']
+    )
+    user.set_password(data['password'])
+    
+    db.session.add(user)
+    db.session.commit()
+    
+    # JWTトークンを生成
+    token = jwt.encode({
+        'user_id': user.id,
+        'exp': datetime.datetime.utcnow() + datetime.timedelta(days=1)
```

Ruff (0.8.2)

`datetime.datetime.utcnow()` used

(DTZ003)